### PR TITLE
feat: add db password route

### PR DESCRIPTION
**Pull request by MutegekiHenry**
### What does this PR do?

- It adds routes for retrieving a DB password.

### How can this be tested?

- Pull branch and checkout the endpoints below:

1. Admin route `/databases/{database_id}/password`
2. User route `/projects/{project_id}/databases/{database_id}/password`

### Link to Trello ticket?

- [https://trello.com/c/H40BqyeK](https://trello.com/c/H40BqyeK)

### CODE REVIEW

**Review by ttcollins — APPROVED** [review]
Nice one @MutegekiHenry

**Review by Masozera — APPROVED** [review]
Hongera @MutegekiHenry .

File: app/routes/__init__.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
     ProjectAppsView, AppDetailView, RegistriesView, ProjectMemoryUsageView, ProjectCPUView, AppMemoryUsageView,
     AppCpuUsageView, AppNetworkUsageView, ProjectNetworkRequestView, AppLogsView, AppStorageUsageView, ProjectStorageUsageView,
     ProjectDatabaseView, ProjectDatabaseDetailView, ProjectDatabaseAdminView, ProjectDatabaseAdminDetailView, 
-    ProjectDatabaseResetView, ProjectDatabaseAdminResetView, ProjectDatabasePasswordResetView, ProjectDatabaseAdminPasswordResetView
-)
+    ProjectDatabaseResetView, ProjectDatabaseAdminResetView, ProjectDatabasePasswordResetView, ProjectDatabaseAdminPasswordResetView, UserDetailView,
```
Comment (Mubangizi) [inline app/routes/__init__.py:18]: Hey @MutegekiHenry, `UserDetailView` is already imported, there is no need to reimport it

**Review by Mubangizi — CHANGES_REQUESTED** [review]
Hey @MutegekiHenry, nice work, but you did not restrict the passwords and they are still being returned when someone calls the database info routes.

**Comment by MutegekiHenry** [comment]
> Hey @MutegekiHenry, nice work, but you did not restrict the passwords and they are still being returned when someone calls the database info routes.

I didn't want to remove them now coz that would break the frontend, wanted them to adjust to this first then we can clean up when they are done.

File: app/routes/__init__.py (hunk 1 above)
Comment (MutegekiHenry) [reply app/routes/__init__.py:18]: This must have been a mistaken import, let me remove it.

Comment (MutegekiHenry) [reply app/routes/__init__.py]: Done sir.

**Review by Mubangizi — APPROVED** [review]
LGTM

**Pull request merged**